cmp/grid/GridModel.js

```diff
@@ -810,6 +810,15 @@ export class GridModel {
 
             if (!isEmpty(agSizable)) {
                 this.agColumnApi?.autoSizeColumns(agSizable);
```

Comment:
We should discuss how import this fallback is, and if is worth the complexity, if there should be a flag for it, etc.

Additional complexity needed here with the wait() brings this question to the fore